StRoot/StETofUtil/StETofGeometry.h

```diff
@@ -387,7 +399,7 @@ inline void  StETofGeometry::debugOn()  { mDebug = true;  }
 inline void  StETofGeometry::debugOff() { mDebug = false; }
 
 inline bool  StETofGeometry::isDebugOn() const { return mDebug; }
+inline void  StETofGeometry::setFileNameAlignParam(std::string FileNameAlignParam) {mFileNameAlignParam = FileNameAlignParam;}
```

Comment:
```suggestion
inline void  StETofGeometry::setFileNameAlignParam(std::string fileNameAlignParam) {mFileNameAlignParam = fileNameAlignParam;}
```

https://star-bnl.github.io/star-coding-guidelines/formatguide.xml#Variable_Names